adoc/chapters/programming_interface.adoc

```diff
@@ -6554,10 +6554,16 @@ yet bound to a <<command-group>>.  Before such an accessor can be used in a
 <<command>>, it must be bound by calling [code]#handler::require()#.  If a
 placeholder accessor is passed as an argument to a <<command>> without first
 being bound to a <<command-group>> with [code]#handler::require()#, the
-implementation throws a synchronous [code]#exception# with the
-[code]#errc::kernel_argument# error code when the <<command>> is submitted.
+implementation may throw a synchronous [code]#exception# with the
+[code]#errc::kernel_argument# error code when the <<command>> is submitted. If
+the implementation does not throw an synchronous [code]#exception# upon
+<<command>> submission, an asynchronous [code]#exception# will be thrown during
```

Comment:
```suggestion
<<command>> submission, an asynchronous [code]#exception# may be thrown during
```
Perhaps we could also add a rationale in a non normative note to avoid too much puzzling.